tools/yoink/templates/metadata.yaml.j2

```diff
@@ -0,0 +1,35 @@
+data:
+  ab_internal:
+    ql: 100
+    sl: 100
+  allowedHosts:
+    hosts:
+      - "{{ host }}"
+  connectorBuildOptions:
+    baseImage: {{ base_image }}
+  connectorSubtype: api
+  connectorType: source
+  definitionId: {{ definition_id }}
+  dockerImageTag: {{ version }}
+  dockerRepository: airbyte/source-{{source_name}}
```

Comment:
💅  Again would remove the inline use of `source-` in favor of a `connector_image_name` property